run-tests.sh

```diff
@@ -2,5 +2,5 @@
 
 # TODO: When we can run docker-compose in Taskcluster, we should use
 # docker-compose-test.yml instead of running docker directly.
-docker build --pull -t balrogtest .
-docker run --rm --entrypoint /app/scripts/test-entrypoint.sh balrogtest $@
+docker build  -t balrogtest -f Dockerfile.dev .
+docker run --rm -v `pwd`:/app --entrypoint /app/scripts/test-entrypoint.sh balrogtest $@
```

Comment:
Using a volume mount here means that ".tox" ends up on the host, which means we don't re-download and re-install all of the test dependencies with every run. This should be a big speed up for those on slower connections.